### DISCUSSION TITLE
Double results?

### DISCUSSION THREAD

**Discussion by telemole** [hlky/stable-diffusion-webui#629] [Q&A]
Just updated my code this morning and my interface is rendering double results (2 of each image created - even though the /outputs folders only show one file... 

![image](https://user-images.githubusercontent.com/22926791/188321653-f2311f0a-7d44-47ab-b1f8-a2aae0cc59bd.png)

Code error, or my mistake somehow?

**Comment by KyuubiYoru** [comment]
Your not on the newest version, this problem was fixed a few days ago.
It should work if you use this repo https://github.com/hlky/stable-diffusion

**Reply by telemole** [reply]
thank-you that's fixed :  I've upgraded and now img2img doesn't work lol

**Reply by HanJammer** [reply]
> Your not on the newest version, this problem was fixed a few days ago. It should work if you use this repo https://github.com/hlky/stable-diffusion

But it wasn't. Previously the previews were not visible to me at all when upscaling was used - no they are visible - one showing non-upscaled and other showing upscaled version. It causes GUI to be very unresponsive as well (I created an Issue ticket).
